spec/ddtrace/configuration_spec.rb

```diff
@@ -259,14 +264,16 @@
           let(:statsd) { instance_double(Datadog::Statsd) }
 
           before do
-            expect(statsd).to_not receive(:close)
+            allow(statsd).to receive(:close)
```

Comment:
Minor: Maybe move the `allow` to be part of the double setup, like the previous test?